filebeat/input/filestream/fswatch.go

```diff
@@ -133,7 +134,7 @@ func (w *fileWatcher) Run(ctx unison.Canceler) {
 }
 
 func (w *fileWatcher) watch(ctx unison.Canceler) {
-	w.log.Info("Start next scan")
+    w.log.Debug("Start next scan")
```

Comment:
```suggestion
	w.log.Debug("Start next scan")
```

This needs to be a tab, not spaces. Let's see if I can fix this with a suggestion.